docs/techpass-status.md

```diff
@@ -5,9 +5,41 @@ This page provides the status of the TechPass service and information on the fol
 <!--- [Previous incidents](#previous-incidents)-->
 
 ## Scheduled maintenance
-Posted on 3 August 2022
 
-**Scheduled** - TechPass portal will be going through a production deployment on 3 August 2022, 5pm. There may be connectivity issues during this time. There are no action required from you.
+### Scheduled for 12 November 2022
+
+We will perform a scheduled maintenance activity on 12 November 2022 between 09:00 and 18:00 SGT and the following services will be unavailable at that time.
+
+**Service(s) unavailable during maintenance**:
+
+- TechPass Portal.
+- TechPass Automation APIs
+
+We will update you when the impacted services are up and operational.
+
+**Activities and user(s) impacted during this maintenance**:
+
+- Unable to provision TechPass accounts for new users using TechPass automation APIs.
+- Unable to provision TechPass or SEED via TechBiz.
+- New users, who are eligible to sign up for TechPass using TechPass Portal, will not be able to do so.
+- Existing TechPass users who are eligible to request for SEED provisioning via the TechPass Portal will not be able to do so.
+- All other tenant admin activities performed on the TechPass Portal will be impacted.
+- Public officers onboarding their devices to SEED will not be able to register their Intune device ID on the TechPass Portal at that time. You may submit the Intune device ID once the maintenance is completed.
+
+**Additional information**:
+Single sign-on to TechPass AAD (TechPass login) will be available during this maintenance. Hence, existing users will be able to access all other SGTS services and GCC 2.0 CMP, as usual, using TechPass.
+
+We apologise for any inconvenience caused. If you require any clarifications on this matter, please drop us an email at [TechPass enquiries](http://enquiries_techpass@tech.gov.sg/).
+Please [create a ticket](https://go.gov.sg/techpass-sr) if you encounter issues other than the mentioned impact. 
+
+*Posted on 31 October 2022*
+
+
+### Scheduled for 03 August 2022
```

Comment:
let's remove this old announcement. so we can keep only the latest announcements here. )